src/main/resources/icon_associations.xml

```diff
@@ -41,12 +41,9 @@
 
         <regex name="Atom" pattern=".*\.atom$" icon="/icons/files/atom.png"/>
         <regex name="AngularJS" pattern=".*angular\.(min\.)?js$" icon="/icons/files/angular.png"/>
-        <regex name="Angular" pattern=".*\.(ng.html|ng.js|module.[tj]s|ng-template)$"
-               icon="/icons/files/angular.png"/>
-        <regex name="Angular Component" pattern=".*\.component.[tj]s$"
-               icon="/icons/files/angularcomponent.png"/>
-        <regex name="Angular Directive" pattern=".*\.directive.[tj]s$"
-               icon="/icons/files/angulardirective.png"/>
+        <regex name="Angular" pattern=".*\.(ng.html|ng.js|module.[tj]s|ng-template)$" icon="/icons/files/angular.png"/>
+        <regex name="Angular Component" pattern=".*\.component.[tj]s$" icon="/icons/files/angularcomponent.png"/>
+        <regex name="Angular Directive" pattern=".*\.directive.[tj]s$" icon="/icons/files/angulardirective.png"/>
```

Comment:
when using a dot (like directive.ts) you need to precede it with \ since it is not inside a grouping parenthese

Reply:
Sorry about that. Is there someplace where I can test these expressions for validity?